<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QComboBox" name="ReportsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>ОТЧЕТЫ</string>
          </property>
          <item>
           <property name="text">
            <string>ПО ИНВЕСТОРАМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ОБЩИЙ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО МАШИНАМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ВОДИТЕЛЯМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ЛОКАЦИЯМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ПОЛЬЗОВАТЕЛЯМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ЗАРЯДКАМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ТИПУ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ДОЛГИ</string>
           </property>
          </item>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QLabel" name="EcoTaxi">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>ECO TAXI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="SettingsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>НАСТРОЙКИ</string>
          </property>
          <item>
           <property name="text">
            <string>ВОДИТЕЛИ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>МАШИНЫ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ИНВЕСТОРЫ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ТИПЫ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ЛОКАЦИИ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПОЛЬЗОВАТЕЛИ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>УДАЛИТЬ ВСЕ</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>310</number>
        </property>
        <property name="rightMargin">
         <number>45</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="dateButton">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>00.00.0000</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>200</height>
           </size>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
          </property>
          <property name="displayFormat">
           <string>hh:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::TimeSpec::LocalTime</enum>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QCheckBox" name="systemTimeCheck">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Время системы</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <property name="spacing">
         <number>50</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop">
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,15">
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>ДОБАВИТЬ СОБЫТИЕ</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QFrame" name="eventFrame">
             <property name="minimumSize">
              <size>
               <width>550</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,0,0">
              <property name="leftMargin">
               <number>30</number>
              </property>
              <property name="topMargin">
               <number>11</number>
              </property>
              <property name="rightMargin">
               <number>30</number>
              </property>
              <property name="bottomMargin">
               <number>11</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="typeLabel">
                  <property name="text">
                   <string>Тип:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="TypesBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="carsLabel">
                  <property name="text">
                   <string>ID машины:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="CarsBox">
                  <property name="placeholderText">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="driversLabel">
                  <property name="text">
                   <string>ID водителя:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="DriversBox">
                  <property name="placeholderText">
                   <string>-</string>
                  </property>
                  <property name="duplicatesEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="sumLabel">
                  <property name="text">
                   <string>Сумма:</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignmentFlag::AlignVCenter">
                 <widget class="QLineEdit" name="SumEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="descLabel">
                  <property name="text">
                   <string>Описание:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="descEdit"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="addEventButton">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>259</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Добавить</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,15,0">
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>ДОБАВИТЬ ЗАРЯДКУ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="chargeFrame">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>30</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="carsLabel_2">
                  <property name="text">
                   <string>ID машины:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="CarsBox_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="driversLabel_2">
                  <property name="text">
                   <string>ID водителя:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="DriversBox_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="locsLabel">
                  <property name="text">
                   <string>Локация:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="LocsBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="kwtLabel">
                  <property name="text">
                   <string>КВТ:</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignmentFlag::AlignVCenter">
                 <widget class="QLineEdit" name="KWTEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="placeholderText">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="timeLabel">
                  <property name="text">
                   <string>Время:</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignmentFlag::AlignVCenter">
                 <widget class="QLineEdit" name="TimeEdit">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="placeholderText">
                   <string>минуты</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item alignment="Qt::AlignmentFlag::AlignHCenter">
               <widget class="QPushButton" name="addChargeButton">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>52</height>
                 </size>
                </property>
                <property name="text">
                 <string>Добавить</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>0.4.17.10</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QComboBox" name="FinesButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>ШТРАФЫ</string>
          </property>
          <item>
           <property name="text">
            <string>ШТРАФЫ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО МАШИНАМ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ПО ВОДИТЕЛЯМ</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="repairsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>РЕМОНТ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eventsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>СОБЫТИЯ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chargesButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>ЗАРЯДКИ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
